packages/google_sign_in/google_sign_in_platform_interface/lib/src/method_channel_google_sign_in.dart

```diff
@@ -30,6 +30,7 @@ class MethodChannelGoogleSignIn extends GoogleSignInPlatform {
       'signInOption': signInOption.toString(),
       'scopes': scopes,
       'hostedDomain': hostedDomain,
+      'clientId': clientId,
```

Comment:
This change need to be published first.

Reply:
@cyanglaz You mean in a separate PR? I can create a new one with only this change

Reply:
Yep, we need this in a separate PR and a version bump for the platform_interface. Then google_sign_in needs to dep on the new version.

Reply:
@cyanglaz I created a new PR https://github.com/flutter/plugins/pull/3686. I didn't added any tests on it, since I suppose the current PR has the necessary tests to cover both PR changes